site/specs/voice.json

```diff
@@ -341,7 +341,7 @@
                     "Calls"
                 ],
                 "summary": "Modify Call",
-                "description": "Interrupts and replaces an active call's BXML document. Two content types may be used: JSON, which allows hanging up the call or triggering a callback to fetch the new BXML; and XML, which allows sending the new BXML immediately.",
+                "description": "Interrupts and replaces an active call's BXML document.",
```

Comment:
This should all continue to be true, we can't break APIs without lots of advance notice.